docs/guide/data_types/maps.rst

```diff
@@ -351,8 +351,9 @@ dictionary. In the following example we change the title of the plot by changing
     import sunpy.data.sample
     import matplotlib.pyplot as plt
     smap = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
-    smap.plot_settings['title'] = "My Second SunPy Plot"
-    smap.plot_settings['cmap'] = plt.cm.Blues_r
+    plot_settings = {}
```

Comment:
This example doesn't seem to be right? What are we recommending people do now? Pass these in as kwargs to `plot`?

Reply:
since there is no `plot_settings` property associated with `sunpy.map.Map`, we have to pass the plot-specific attributes like `title` as kwargs to `plot`.
I just realized this can have a better implementation as below:
```python
plot_settings = {'title':'My Second SunPy Plot'}
smap.cmap = plt.cm.Blues_r
fig = plt.figure()
smap.plot()
```

Reply:
My point was you aren't actually passing the `plot_settings` anywhere after you create it. What I think we should be documenting is:
```python
fig = plt.figure()
smap.plot(cmap="Blues_r", title="My Second Plot")
```
and not constructing `plot_settings` or setting `map.norm` / `map.cmap` etc

Reply:
oops sorry, didn't notice I wasn't passing `plot_settings` anywhere.
Can't we pass `cmap` as property of `map`?
```python
smap.cmap = plt.cm.Blues_r
fig = plt.figure()
smap.plot(title="My Second Plot")
````
In this way user can adhere to the recent API as well